pkg/cmdutil/wording/synonyms.go

```diff
@@ -0,0 +1,67 @@
+package wording
+
+import (
+	"fmt"
+	"strings"
+
+	validator "github.com/algolia/cli/pkg/cmdutil/validators"
+	"github.com/algolia/cli/pkg/utils"
+)
+
+func GetSynonymSuccessWording(opts validator.SaveOptions, saveWording string) string {
```

Comment:
I didn't add tests to this function since it's already tested in `synonyms_test.go`

Reply:
After reflexion, every function should be unit tested!